src/groovy/au/org/emii/aatams/JodaOverrides.groovy

```diff
@@ -0,0 +1,42 @@
+package au.org.emii.aatams
+
+import org.joda.time.Interval
+import org.joda.time.base.AbstractInterval
+
+class JodaOverrides {
+
+    // Would be nicer to do this with an "Extension Module", but alas, that's
+    // only available in groovy 2.0.
+    //
+    // An extension module would avoid the need to mock/unmock in every test
+    // which executes the overlapsAny code.
+    static void apply() {
+        AbstractInterval.metaClass.contains = { List instantList ->
```

Comment:
That's what I had originally called it, but then thought just overloading the method would make for a nice API (see: http://joda-time.sourceforge.net/apidocs/org/joda/time/base/AbstractInterval.html - that method is already overloaded).

I could go either way.